protocol.html

```diff
@@ -716,28 +716,56 @@ <h2 property="schema:name">Notification Features</h2>
+              </ul>
+
+              <p>Implementations can augment the <a href="https://www.w3.org/TR/json-ld11/#the-context">JSON-LD Context</a> definition and extend the content of the notifications. See for <a href="https://www.w3.org/TR/activitystreams-core/#example-using-multiple-vocabularies">example using multiple vocabularies</a>. The following examples use JSON-LD serializations with <code>https://www.w3.org/ns/activitystreams</code> and <code>https://www.w3.org/ns/solid/notification/v1</code> as the JSON-LD Context.</p>
+
+              <figure id="notification-update" class="example listing" rel="schema:hasPart" resource="#notification-update">
+                <p class="example-h"><span>Example</span>: An update activity</p>
+
+                <pre about="#notification-update" property="schema:description" typeof="fabio:Script"><code>{</code>
 <code>  "@context": [</code>
 <code>    "https://www.w3.org/ns/activitystreams",</code>
 <code>    "https://www.w3.org/ns/solid/notification/v1"</code>
 <code>  ],</code>
-<code>  "id":"urn:uuid:&lt;uuid&gt;",</code>
-<code>  "type": [</code>
-<code>    "Update"</code>
-<code>  ],</code>
-<code>  "object": {</code>
-<code>    "id":"https://example.org/&lt;user&gt;/inbox/",</code>
-<code>     "type": [</code>
-<code>      "http://www.w3.org/ns/ldp#BasicContainer"</code>
-<code>    ]</code>
-<code>  },</code>
+<code>  "id": "urn:uuid:fc8b5af4-bd7e-4fd1-a649-afcbd0e1c083",</code>
+<code>  "type": "Update",</code>
+<code>  "object": "https://example.org/guinan/profile",</code>
 <code>  "state": "1234-5678-90ab-cdef-12345678",</code>
-<code>  "published":"2021-08-05T01:01:49.550044Z"</code>
+<code>  "published": "2021-08-05T01:01:49.550Z"</code>
+<code>}</code></pre>
+
+                <figcaption property="schema:name">An activity indicating that an object (profile) was updated.</figcaption>
+              </figure>
+
+              <figure id="notification-add" class="example listing" rel="schema:hasPart" resource="#notification-add">
+                <p class="example-h"><span>Example</span>: An add activity</p>
+
+                <pre about="#notification-add" property="schema:description" typeof="fabio:Script"><code>{</code>
+<code>  "@context": [</code>
+<code>    "https://www.w3.org/ns/activitystreams",</code>
+<code>    "https://www.w3.org/ns/solid/notification/v1"</code>
+<code>  ],</code>
+<code>  "id": "urn:uuid:a4da6738-6be0-11ed-90bd-1be4ba6b6b33",</code>
+<code>  "type": "Add",</code>
+<code>  "object": "https://example.org/guinan/inbox/notification",</code>
+<code>  "target": "https://example.org/guinan/inbox/",</code>
+<code>  "published": "2022-11-24T11:55:31.483Z"</code>
 <code>}</code></pre>
 
-              <p>Implementations can augment the JSON-LD context definition and extend the content of the notifications.</p>
+                <figcaption property="schema:name">An activity indicating that an object (notification) was added to the target (inbox).</figcaption>
```

Comment:
I wouldn't use example with notification about notification. While they could be used this way I think it adds possibility of confusion without adding any value. @csarven could you change the example to adding anything else than a notification to a container? A calendar event, a photo to the album, a card to the address book, a song to a playlist, anything will do here.

Reply:
Okay with me https://github.com/solid/notifications/pull/124/commits/c9a5885a206c1597a0f2eef1f2660b78e1c8f016 . Thanks.